plugins/modules/fabric_virtual_networks_workflow_manager.py

```diff
@@ -0,0 +1,2950 @@
+    ccc_virtual_network.sda.add_layer3_virtual_networks
+    ccc_virtual_network.sda.update_layer3_virtual_networks
+    ccc_virtual_network.sda.delete_layer3_virtual_network_by_id
+    ccc_virtual_network.sda.get_reserve_ip_subpool
+    ccc_virtual_network.sda.get_anycast_gateways
+    ccc_virtual_network.sda.add_anycast_gateways
+    ccc_virtual_network.sda.update_anycast_gateways
+    ccc_virtual_network.sda.delete_anycast_gateway_by_id
+
+"""
+
+EXAMPLES = r"""
+- name: Create layer2 fabric Vlan for sda in Cisco Catalyst Center.
+  cisco.dnac.fabric_virtual_networks_workflow_manager:
+    dnac_host: "{{dnac_host}}"
+    dnac_username: "{{dnac_username}}"
+    dnac_password: "{{dnac_password}}"
+    dnac_verify: "{{dnac_verify}}"
+    dnac_port: "{{dnac_port}}"
+    dnac_version: "{{dnac_version}}"
+    dnac_debug: "{{dnac_debug}}"
+    dnac_log_level: "{{dnac_log_level}}"
+    dnac_log: False
+    state: merged
+    config:
+      - fabric_vlan:
+        - vlan_name: "vlan_test1"
+          fabric_site_locations:
+          - site_name_hierarchy: "Global/India"
+            fabric_type: "fabric_site"
+          - site_name_hierarchy: "Global/India/Chennai"
+            fabric_type: "fabric_zone"
+          vlan_id: 1333
+          traffic_type: "DATA"
+          fabric_enabled_wireless: False
+        - vlan_name: "vlan_test2"
+          fabric_site_locations:
+          - site_name_hierarchy: "Global/USA"
+            fabric_type: "fabric_site"
+          vlan_id: 1334
+          traffic_type: "VOICE"
+          fabric_enabled_wireless: False
+
+- name: Update layer2 fabric Vlan for sda in Cisco Catalyst Center.
+  cisco.dnac.fabric_virtual_networks_workflow_manager:
+    dnac_host: "{{dnac_host}}"
+    dnac_username: "{{dnac_username}}"
+    dnac_password: "{{dnac_password}}"
+    dnac_verify: "{{dnac_verify}}"
+    dnac_port: "{{dnac_port}}"
+    dnac_version: "{{dnac_version}}"
+    dnac_debug: "{{dnac_debug}}"
+    dnac_log_level: "{{dnac_log_level}}"
+    dnac_log: False
+    state: merged
+    config:
+      - fabric_vlan:
+        - vlan_name: "vlan_test1"
+          fabric_site_locations:
+          - site_name_hierarchy: "Global/India"
+            fabric_type: "fabric_site"
+          - site_name_hierarchy: "Global/India/Chennai"
+            fabric_type: "fabric_zone"
+          vlan_id: 1333
+          traffic_type: "VOICE"
+          fabric_enabled_wireless: True
+
+- name: Deleting layer2 fabric Vlan from the Cisco Catalyst Center.
+  cisco.dnac.fabric_virtual_networks_workflow_manager:
+    dnac_host: "{{dnac_host}}"
+    dnac_username: "{{dnac_username}}"
+    dnac_password: "{{dnac_password}}"
+    dnac_verify: "{{dnac_verify}}"
+    dnac_port: "{{dnac_port}}"
+    dnac_version: "{{dnac_version}}"
+    dnac_debug: "{{dnac_debug}}"
+    dnac_log_level: "{{dnac_log_level}}"
+    dnac_log: False
+    state: deleted
+    config:
+      - fabric_vlan:
+        - vlan_name: "vlan_test1"
```

Comment:
Do we need to give all information to delete? Can you please give whatever is required to delete?